mittab/apps/tab/models.py

```diff
@@ -3,21 +3,16 @@
 from haikunator import Haikunator
 from django.db import models
 from django.core.exceptions import ValidationError
-from polymorphic.models import PolymorphicModel
 
+from mittab.libs import cache_logic
 
-class ModelWithTiebreaker(PolymorphicModel):
```

Comment:
For some reason django-polymorphic introduces some bad behavior with prefetch_related, since it's such a minor thing here I chose to just remove the dependency